_posts/en/newsletters/2021-01-20-newsletter.md

```diff
@@ -0,0 +1,125 @@
+permalink: /en/newsletters/2021/01/20/
+name: 2021-01-20-newsletter
+slug: 2021-01-20-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes posts to the Bitcoin-Dev mailing list
+about payjoin adoption and making hardware wallets compatible with more
+advanced Bitcoin features.  Also included are our regular sections with
+overviews of changes to services and client software, new releases and
+release candidates, and changes to popular Bitcoin infrastructure
+software.
+
+## News
+
+- **Payjoin adoption:** Chris Belcher [posted][belcher payjoin] to the
+  Bitcoin-Dev mailing list a request for people to look for ways to
+  increase [payjoin][topic payjoin] adoption along with a [wiki
+  page][payjoin wiki] tracking the projects that provide either sending
+  or receiving support for payjoin.  One [suggestion][raw payjoin] by
+  Craig Raw was to extend the protocol to allow it to work even when the
+  receiver doesn't operate a server.
+
+- **Making hardware wallets compatible with more advanced Bitcoin features:**
+  Kevin Loaec [started a discussion][loaec hww] on the Bitcoin-Dev
+  mailing list about how hardware wallets could be changed to allow them
+  to handle scripts more complicated than single-sig or multisig.  For
+  example, allowing a hardware wallet to handle in-channel LN payments
+  or payments made from a [vault][topic vaults].  His post does a good
+  job of describing various problems that current hardware wallets can't
+  handle, but he notes that necessary "changes may be very difficult".
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday
+
+- [Bitcoin Core 0.21.0][Bitcoin Core 0.21.0] is a the next major version
+  of this full node implementation and its associated wallet and other
+  software.  Major new features include support for Tor version 3 hidden
+  services using [version 2 `addr` messages][topic addr v2], the optional
+  ability to serve [compact block filters][topic compact block filters],
+  support for [signets][topic signet] (including the default signet
+  which has [taproot][topic taproot] activated).  It also offers
+  experimental support for wallets that natively use [output script
+  descriptors][topic descriptors].  For a complete list of changes, see
+  the [release notes][bcc 0.21.0 notes].
+
+- [Rust Bitcoin 0.26.0][] is a new release of this library.  Major new
+  features include support for signet, version 2 `addr` messages,
+  and several improvements to [PSBT][topic psbt] handling.  See its
+  [release notes][rb notes] for details.
+
+- [BTCPay Server 1.0.6.7][] in the second maintenance release to
```

Comment:
```suggestion
- [BTCPay Server 1.0.6.7][] is the second maintenance release to
```

"the second release to a release" construction seems awkward

Reply:
Yeah, it is awkward.  The problem is that 1.0.6.7 is an uninteresting bug fix release and that 1.0.6.5 is the one that adds features I think are interesting.  However, I think it'd be a bad idea to start the bullet with a link to a version I know has bugs, so I'm kinda stuck with this awkward construction.  (Suggestions welcome, of course.)

As a side note, BTCPay Server seems to put out a lot of small bug fix releases.  I haven't used their software myself, but I'm guessing they have an auto-update tool that makes that reasonable in a way it isn't for the other manually-updated software we usually cover.  As a result of this, I don't think I'm going to cover every BTCPay release in this section, just the ones that seem significant to me.  Let me know if you see a problem with that.

Reply:
How about:

> Last week, BTCPay Server [1.0.6.5][btcpay server 1.0.6.5] added "support [for] a subset of output descriptors in the wallet setup" (see the  *notable changes* section below). Since then, two maintenance releases added features and bug fixes which makes [BTCPay Server 1.0.6.7][] the latest version at the time of writing.

Reply:
> As a result of this, I don't think I'm going to cover every BTCPay release in this section, just the ones that seem significant to me. Let me know if you see a problem with that.

Seems totally reasonable!

Reply:
Marking this as resolved. Also pushed a small commit to s/in/is.